src/Akka.Cluster.Hosting/ClusterClientReceptionistOptions.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System;
+using System.Runtime.CompilerServices;
+using System.Text;
+using System.Text.RegularExpressions;
+using Akka.Configuration;
+
+namespace Akka.Cluster.Hosting
+{
+    public class ClusterClientReceptionistOptions
+    {
+        /// <summary>
+        /// Actor name of the ClusterReceptionist actor, /system/receptionist
+        /// </summary>
+        public string Name { get; set; } = "receptionist";
+
+        /// <summary>
+        /// Start the receptionist on members tagged with this role. All members are used if undefined.
+        /// </summary>
+        public string Role { get; set; }
+
+        /// <summary>
+        /// The receptionist will send this number of contact points to the client.
+        /// </summary>
+        public int? NumberOfContacts { get; set; }
+
+        /// <summary>
+        /// The actor that tunnel response messages to the client will be stopped after this time of inactivity.
+        /// </summary>
+        public TimeSpan? ResponseTunnelReceiveTimeout { get; set; }
+
+        /// <summary>
+        /// How often failure detection heartbeat messages should be received for each ClusterClient
+        /// </summary>
+        public TimeSpan? HeartbeatInterval { get; set; }
+
+        /// <summary>
+        /// Number of potentially lost/delayed heartbeats that will be
+        /// accepted before considering it to be an anomaly.
+        /// The ClusterReceptionist is using the akka.remote.DeadlineFailureDetector, which
+        /// will trigger if there are no heartbeats within the duration
+        /// heartbeat-interval + acceptable-heartbeat-pause, i.e. 15 seconds with
+        /// the default settings.
+        /// </summary>
+        public TimeSpan? AcceptableHeartbeatPause { get; set; }
+
+        /// <summary>
+        /// Failure detection checking interval for checking all ClusterClients
+        /// </summary>
+        public TimeSpan? FailureDetectionInterval { get; set; }
```

Comment:
Same feedback as above - I'd leave it as just Name, Role, NumberOfContracts, and ResponseTunnelReceiveTimeout.

On the last property I'd make it clear that this is optional and defaults to 60 seconds when not set.